modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage on the command line.
+
+.Prerequisites
+
+* You have installed the latest version of the OpenShift CLI. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You are ready on {ibm-cloud-title}. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
+
+** You have an {ibm-cloud-title} Platform account.
+** You have ordered an {ibm-cloud-title} Object Storage plan.
+** You have created an instance of {ibm-cloud-title} Object Storage.
+
+.Procedure
+
+. On {ibm-cloud-title}, create an object store bucket.
+
+. On {ibm-cloud-title}, create a service key with the following parameters for connecting to the object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud resource service-key-create <tempo_bucket> Writer --instance-name <tempo_bucket> --parameters '{"HMAC":true}'
+----
+
+. On {ibm-cloud-title}, create a secret with the bucket credentials:
++
+[source,terminal]
+----
+$ oc -n <namespace> create secret generic <ibm_cos_secret> \
+  --from-literal=bucket="<tempo_bucket>" \
+  --from-literal=endpoint="<ibm_bucket_endpoint>" \
+  --from-literal=access_key_id="<ibm_bucket_access_key>" \
+  --from-literal=access_key_secret="<ibm_bucket_secret_key>"
+----
+
+. On the {product-title}, create an object storage secret with keys as follows:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <ibm_cos_secret>
+stringData:
+  bucket: <tempo_bucket>
+  endpoint: <ibm_bucket_endpoint>
+  access_key_id: <ibm_bucket_access_key>
+  access_key_secret: <ibm_bucket_secret_key>
+type: Opaque
+----
+
+. On the {product-title}, set the storage section in the TempoStack custom resource:
```

Comment:
```suggestion
. On the {product-title}, set the storage section in the `TempoStack` custom resource:
```

Reply:
I think references to the CR should be in backticks.

Reply:
Thank you for catching this. Applied.